src/backend/generator.js

```diff
@@ -61,6 +61,15 @@ handlebars.registerHelper('ifvalue', function (conditional, options) {
     }
 });
 
+handlebars.registerHelper('checkpage', function (conditional, options) {
+    if (conditional == options.hash.equals){
```

Comment:
Please add some comments for this helper to understand this better
Also use === instead of `==`